demisto_sdk/__main__.py

```diff
@@ -458,6 +458,14 @@ def zip_packs(**kwargs) -> int:
     help="The path of the id-set.json used for validations.",
     type=click.Path(resolve_path=True),
 )
+@click.option(
+    "-vg",
+    "--validate-graph",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Perform validations on content graph.",
+)
```

Comment:
`--validate-graph` suggests we validate the graph itself - is this what we do here?

Reply:
I suggest changing it to `--graph`, `--use-graph` or `--use-content-graph` (a bit long), as we validate the _content_ using the graph

Reply:
Decided to use `-gr` and `--graph` to simplify things